tests/RobotFramework/tasks/E2E_Tutorials/connect_to_c8y_send_measurements.robot

```diff
@@ -0,0 +1,192 @@
+
+Send Combined Measurement
+    [Documentation]    Send a combined measurement.
+    Execute Command    tedge mqtt pub 'te/device/main///m/combined' '{"time": "2020-10-15T05:30:47+00:00", "temperature": 25, "current": {"L1": 9.5, "L2": 10.3, "L3": 8.8}, "pressure": 98}'
+    Log    Sent combined measurement
+
+Send Child Device Temperature Measurement
+    [Documentation]    Send a temperature measurement to a child device.
+    Execute Command    tedge mqtt pub 'te/device/child1///m/environment' '{"temperature": 25}'
+    Log    Sent temperature measurement to child device: 25°C
+
+Verify Measurement In Cumulocity
+    [Arguments]        ${type} 
+    Device Should Have Measurements    type=${type}    minimum=1    maximum=1
+
+Verify Combined Measurement In Cumulocity
+    [Documentation]    Verify the combined measurement in Cumulocity.
+    # Add implementation for checking the combined measurement in Cumulocity
+    Log    Verified combined measurement in Cumulocity
+
+Verify Child Device Measurement In Cumulocity
+    [Arguments]    ${child_id}    ${type}    ${value}
+    [Documentation]    Verify the child device measurement in Cumulocity.
+    # Add implementation for checking the child device measurement in Cumulocity
+    Log    Verified ${type} measurement for child device ${child_id} with value ${value} in Cumulocity
+
+
+Custom Setup
+    [Documentation]    Initializes the device environment. 
+    ...                Sets up the device, transfers necessary packages, 
+    ...                installs them, and configures Cumulocity for connectivity.
+    ${DEVICE_SN}=    Setup    skip_bootstrap=True
+    Set Suite Variable    ${DEVICE_SN}
+
+    # Determine the device architecture
+    ${output}    Execute Command    uname -m
+    ${arch}    Set Variable    ${output.strip()}
+
+    # Conditional file transfer based on architecture
+    Run Keyword If    '${arch}' == 'aarch64'    Transfer Aarch64 Packages
+    ...    ELSE IF    '${arch}' == 'armv7l'    Transfer Armv7l Packages
+    ...    ELSE    Log    Unsupported architecture: ${arch}
+
+    # Install packages
+    Execute Command    sudo dpkg -i *.deb
+    Log    Installed new packages on device
+
+Transfer Aarch64 Packages
+    [Documentation]    Transfers Aarch64 architecture packages to the device.
+    ${log}    Transfer To Device    target/aarch64-unknown-linux-musl/packages/*.deb    /home/pi/
+    Log    Transferred Aarch64 packages to device
+
+Transfer Armv7l Packages
+    [Documentation]    Transfers ARMv7l architecture packages to the device.
+    ${log}    Transfer To Device    target/armv7-unknown-linux-musleabihf/packages/*.deb    /home/pi/
+    Log    Transferred ARMv7l packages to device
+
+Custom Teardown
+    [Documentation]    Cleans up the device environment. 
+    ...                Uninstalls ThinEdgeIO, removes packages and scripts, and retrieves logs.
+    Transfer To Device    ${CURDIR}/uninstall-thin-edge_io.sh    /home/pi/uninstall-thin-edge_io.sh
```

Comment:
```suggestion
    Transfer To Device    ${CURDIR}/uninstall-thin-edge_io.sh    /var/local/share/uninstall-thin-edge_io.sh
```

Reply:
Though there are other places which also use the `/home/pi` folder...best to avoid this all together.